proto/expr.proto

```diff
@@ -161,8 +161,10 @@ message TableFunction {
   data.DataType return_type = 3;
 }
 
-message InputRefExpr {
-  int32 column_idx = 1;
+// Reference to a column, containing its index and data type.
+message ColumnRef {
```

Comment:
prefer `InputRef`. sometimes someone mixed up the input_col_idx and output_col_idx.